rpcs3/Emu/system_config.h

```diff
@@ -295,7 +295,8 @@ struct cfg_root : cfg::node
 		cfg::_enum<CellKbMappingType> keyboard_type{ this, "Keyboard Type", CellKbMappingType{0} }; // CELL_KB_MAPPING_101 = US
 		cfg::_enum<enter_button_assign> enter_button_assignment{ this, "Enter button assignment", enter_button_assign::cross };
 		cfg::_int<-60*60*24*365*100LL, 60*60*24*365*100LL> console_time_offset{ this, "Console time offset (s)", 0 }; // console time offset, limited to +/-100years
-
+                cfg::string console_psid_high{ this, "PSID high", "0x0"};
+                cfg::string console_psid_low{ this, "PSID low", "0x0"};
```

Comment:
Why use string instead of integer?

Reply:
you have examples of it above.

Reply:
Just as thought that would make it easier for a gui string box?

Reply:
Maybe but the additional logic for it there is acceptable because of the builtin error checking for values and improved usage performance in actual execution (in case the syscall is spammed) are more importabt

Reply:
> (and so it gets saved as a hex string too rather than getting converted to dec)
Integer settings allow hex values, even negative hex. I think what's missing is default hex saving entry type, but that can be implemented later.

Reply:
On second thought it doesn't really matter for GUI either and is a very specific QoL feature for manual config file reading which can be implemented at any time and for other settings as well.

Reply:
well that makes it every easier (and i see saving the config file doesn't alter unchanged settings anyway)